GNUmakefile

```diff
@@ -79,7 +79,7 @@ presubmit: ## Format go files and imports.
 
 .PHONY: mockery
 mockery: $(mockery) ## Install mockery.
-	go install github.com/vektra/mockery/v2@v2.10.1
+	go install github.com/vektra/mockery/v2@v2.10.4
```

Comment:
https://github.com/vektra/mockery#note-on-golang-118
> **NOTE ON Golang 1.18**
Mockery currently does not support various 1.18 Golang features, namely generics. PRs are welcome!

We weren't doing any complex behavioral mocking with these ORMs anyways, so it was trivial to replace the few usages with partial in-mem impls.

Reply:
Is this mockery version supposed to match the one in the .tool-versions file?